src/main/deploy/pathplanner/autos/New Auto.auto

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Should this and the other pathplanner files be checked in? This one seems empty, and the other two are maybe just local user-state items?